src/arkode/arkode_mristep_nls.c

```diff
@@ -355,7 +360,7 @@ int mriStep_Nls(ARKodeMem ark_mem, int nflag)
   This routine wraps the ARKODE linear solver interface 'setup'
   routine for use by the nonlinear solver object.
   ---------------------------------------------------------------*/
-int mriStep_NlsLSetup(booleantype jbad, booleantype* jcur, void* arkode_mem)
+SUNNlsStatus mriStep_NlsLSetup(booleantype jbad, booleantype* jcur, void* arkode_mem)
```

Comment:
The return values inside this function are not valid `SUNNlsStatus` values (e.g., `ARK_LSETUP_FAIL`, `CONV_FAIL`, `ARK_SUCCESS`); each should probably be converted to a named `SUNNlsStatus` constant.

Reply:
Good point. Will fix.